docs/source/en/guides/collections.md

```diff
@@ -0,0 +1,187 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# Manage your collections
+
+A collection is a group of related items on the Hub (models, datasets, Spaces, papers) that are organized together on a same page. Collections can be useful in many use cases such as creating your own portfolio, bookmarking content in categories or presenting a curated list of items your want to share. Check out this [guide](https://huggingface.co/docs/hub/collections) to understand in more details what are Collections and how they look like on the Hub,
+
+Managing collections can be done in the browser directly. In this guide, we will focus on how to do it programmatically using `huggingface_hub`.
+
+## Fetch a collection
+
+To fetch a collection, use [`get_collection`]. You can use it either on your own collections or any public one. To retrieve a collection, you must have its collection's `slug`. A slug is an identifier for a collection based on the title and a unique ID. You can find it in the URL of the collection page.
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hfh_collection_slug.png"/>
```

Comment:
Minor detail, but I think it'd be nice to highlight/underline the slug in the URL to immediately draw the user's eyes to it.

Reply:
Good idea. I updated the remote image: https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hfh_collection_slug.png